### DISCUSSION TITLE
Aspect type "Tooling"

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#591] [Ideas] [answered]
I would like to expend the list of Aspects types with the type "technology" (for the classification of tooling and / or links to github coding)

**Comment by techsmyth** [comment, marked as answer]
Hi Gerlof, the set of aspect types can be updated per hub. So if you send through the aspect type ("tooling"), a description for the aspect type ("relevenat tools / approaches"?) and what default text you want when the user adds the description (main body text default) then we can add it...

Cheers, Neil

**Reply by techsmyth** [reply]
Note that this is not yet customizable via the Admin UI, but is something we are working towards.

**Reply by GerlofdeHaan** [reply]
Hi Neil,

Ok. For now I think this would work:

Type: "techniek"
Description: "relevante technische oplossingen"
Default text: "beschrijf hier de relevante technische oplossing"
